src/display/editor/editor.js

```diff
@@ -921,6 +926,17 @@ class AnnotationEditor {
     }
   }
 
+  #enableAltTextButton(enabled = false) {
```

Comment:
Naming things is always difficult, but having a method whose name starts with "enable" and its default value then being to *disable* something feels a little confusing. How about the following name instead?
```suggestion
  #toggleAltTextButton(enabled = false) {
```